cmake/partition_manager.cmake

```diff
@@ -610,6 +610,15 @@ to the external flash")
     ${global_hex_depends}
     )
 
+  add_custom_target(
+    partition_manager_report
+    COMMAND
+    ${PYTHON_EXECUTABLE}
+    ${ZEPHYR_BASE}/../nrf/scripts/partition_manager_report.py
```

Comment:
I know this line already exists, but I'm just noticing it now, so I'll comment here without trying to block this PR. If you have time to fix it, great.

It seems odd to get the location of the sdk-nrf repository starting from zephyr in the workspace.

This would actually break if the user workflow has something like this as the west manifest file:

```
manifest:
  projects:
    - name: zephyr
      url: https://github.com/mycompany/my-zephyr
    - name: nrf
      path-prefix: imported-from-ncs
      import: true
```

Because then sdk-nrf is in `imported-from-ncs/nrf` and zephyr is in `zephyr`.

I tested using `${CMAKE_CURRENT_LIST_DIR}/../scripts/partition_manager_report.py` locally and it worked fine; why not do that instead?

Reply:
I changed to use `ZEPHYR_MODULE_NRF_DIR` which seems to be the idiomatic way of doing it.